app/lib/registries/full_topic_taxonomy_registry.rb

```diff
@@ -0,0 +1,57 @@
+module Registries
+  class FullTopicTaxonomyRegistry
+    CACHE_KEY = "#{NAMESPACE}/full_topic_taxonomy".freeze
+
+    def [](base_path)
+      taxonomy_tree[base_path]
+    end
+
+    def taxonomy_tree
```

Comment:
This name isn't accurate because it's not a tree

Reply:
Thanks - I'll rename